homeassistant/components/google_cloud/tts.py

```diff
@@ -239,7 +239,7 @@ async def async_get_tts_audio(self, message, language, options=None):
 
         try:
             # pylint: disable=no-member
-            synthesis_input = texttospeech.types.SynthesisInput(text=message)
+            synthesis_input = texttospeech.types.SynthesisInput(ssml=message)
```

Comment:
This is not correct. SSML is an XML based standard and text is not valid XML.

https://docs.microsoft.com/en-us/cortana/skills/speech-synthesis-markup-language#speak-element

Reply:
I guess it uses Google's version of ssml, here is the documentation: https://cloud.google.com/text-to-speech/docs/ssml

You can take a look for yourself, if you want I can provide you with the custom component and a sample to test this.

Reply:
Google uses the same spec, and it says that `<speak>` is the root element: https://cloud.google.com/text-to-speech/docs/ssml#speak

Reply:
It works without the <speak> tag, I guess changing the code to ssml makes the <speak> tag not needed, but other syntax works